### DISCUSSION TITLE
🐸 TTS Community Meeting v0.5

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#957] [Announcements]
Here are the specs for our 🐸 community meeting v0.5 ✨

We plan to answer live questions then jump to the link below in the order of votes. Feel free to post your questions starting from today.

We'll post the video recording somewhere in case  you miss the call

So let's see how it's gonna turn out 😄

👉  Meeting link

https://meet.google.com/jys-yfpv-wyd

👉  Meeting time

Dec 2, 2021 Thursday 17:30 - 18:15 CET
(Add to your calendar from [here](https://cutt.ly/kTHFP3Q))

👉  Ask or upvote questions here

https://github.com/coqui-ai/TTS/discussions/categories/qs-for-live-q-a

**Comment by erogol** [comment]
We postpone our event for a week not to conflict with Thanksgiving :)

I updated the date above accordingly

**Reply by michaellin99999** [reply]
where can we find the video?

**Comment by loganhart02** [comment]
https://youtu.be/2yup-Eeebq0
